pkg/mqtt5/interface.go

```diff
@@ -26,4 +27,8 @@ type MessageClient interface {
 
 	// Unsubscribe to unsubscribe from the specified topics.
 	Unsubscribe(topics []string) error
+
+	// Publish is to send message to the message bus
+	// the message contains data payload to send to the message queue
+	Publish(topic string, message models.MessageEnvelope) error
```

Comment:
use the errors.Error interface for all the error type in this interface
maybe open another Jira ticket and modify in the next PR

Reply:
updated